### ISSUE TITLE
Convert all parts from educational licence to commercial license

### ISSUE THREAD

**Issue by marcellosavoldi** [icub-tech-iit/cad-libraries#4]
Some ideas?

**Comment by divyashah** [comment]
For the new parts that we add to `cad-libraries` should we convert the license manually ?? 
Or do we have any updates on having a script to convert the licenses ?? 

cc @fiorisi  @marcellosavoldi

**Comment by marcellosavoldi** [comment]
eh... good question...
I forgot this issue ...We need to investigate!
@fiorisi ...How we made for the cad-libraries-pubblic ?

**Comment by fiorisi** [comment]
> For the new parts that we add to cad-libraries should we convert the license manually ??

Yes @divyashah, for the time being we have to do it manually for the files that we manipulate/create.
